### PR TITLE
Removed SfBO Connector and move public preview instructions to the end (teams-powershell-install.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6645.

@tseward @get-itips, what do you think guys? Should we remove SfBO connector? Thanks!

@nam31, could you add them for review? Thanks!

### CODE REVIEW

**Comment by get-itips** [comment]
I agree

**Comment by nam31** [comment]
@JohanFreelancer9, please do the copy editor review for this PR.

**Review by get-itips — APPROVED** [review]

File: Teams/teams-powershell-install.md [hunk 1]
```diff
@@ -134,6 +94,29 @@ Uninstall-Module MicrosoftTeams
 > [!WARNING]
 > If Teams PowerShell has already been imported into your PowerShell session, uninstalling the module will fail. Close PowerShell and re-open a new elevated PowerShell session.
 
+## Install Teams PowerShell public preview
+
+> [!NOTE]
+> If you're using the Public Preview version of Teams PowerShell, we strongly recommend that you first uninstall Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:100]:
```suggestion
> If you're using the public preview version of Teams PowerShell, we strongly recommend that you first uninstall Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by nam31** [comment]
@brandber, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by nam31** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@cichur PR has been copyedited and is ready for final review. Could you please check and merge? Thanks!

**Pull request merged**